### PR TITLE
Form issues 103

### PR DESCRIPTION
**Pull request by lboeman**
Various minor form issues fixed for 1.0.3. Commit messages include issues closed.

### CODE REVIEW

**Comment by lboeman** [comment]
@wholmgren do you have time for a quick look at this one? Second set of eyes to check for typos and awkward grammar would be appreciated.

File: sfa_dash/templates/forms/site_form.html [hunk 1]
```diff
@@ -110,7 +112,8 @@
       </div>
       <div class="form-element">
         {{ form.input('Ground coverage ratio', 'number', 'ground_coverage_ratio',
-                      help_text='Azimuth angle of the tracker axis, degrees East of North.', form_data=form_data,
+                      help_text='Ratio of total width of modules on a tracker to the distance between tracker axes. For example, for trackers each with two modules of 1m width each, and a spacing between tracker axes of 7m, the ground coverage ratio is 0.286(=2/7).',
```
Comment (wholmgren) [inline sfa_dash/templates/forms/site_form.html:115]:
```suggestion
                      help_text='Ratio of total width of modules on a tracker to the distance between tracker axes. For example, for trackers with two modules of 1m width, and a spacing between tracker axes of 7m, the ground coverage ratio is 0.286(=2/7).',
```

Comment (wholmgren) [reply sfa_dash/templates/forms/site_form.html:115]: Debatable but I think it makes more sense without each each

**Review by wholmgren — APPROVED** [review]
🥇 for number of issues closed in one pr.

File: sfa_dash/templates/forms/site_form.html (hunk 1 above)
Comment (lboeman) [reply sfa_dash/templates/forms/site_form.html:115]: Agreed.

**Pull request merged**